src/components/BlogPage.jsx

```diff
@@ -97,6 +97,34 @@ const BlogPage = (props) => {
     }
   };
 
+  const compareBy = (key) => {
+    return function (a, b) {
+      if (a[key].toLowerCase() < b[key].toLowerCase()) return -1;
+      if (a[key].toLowerCase() > b[key].toLowerCase()) return 1;
+      return 0;
+    };
+  };
+
+  const sortPosts = (e) => {
+    let arrayCopy = [...posts];
+    console.log("SORT POSTS, e.target.value: " + e.target.value);
+    if (e.target.value === "titleAsc") {
```

Comment:
@marijana-kolosek Try refactor this more. I think you can move setFilteredPosts([...arrayCopy]) to the bottom.